### DISCUSSION TITLE
Configuration error again and again how do i solve ?

### DISCUSSION THREAD

**Discussion by sahil3276** [JanDeDobbeleer/oh-my-posh#4776] [General]
![git](https://github.com/JanDeDobbeleer/oh-my-posh/assets/105536637/9aa88234-d7e5-4678-9ed4-0f3998c5b5fe)
im new to github.. i dont know how to raise query if i raise this in wrong form please pardon me.. can you help me how do i solve it.. i follow one youtube video step by step but i am not able to get the theme..

**Comment by JanDeDobbeleer** [comment]
Couple of things. You can see the actual error when using `oh-my-posh debug` that one has all the details on what's going on. Secondly, you want to install and configure a nerd font in your terminal.
